cpp/src/arrow/util/bit_util_test.cc

```diff
@@ -1120,22 +1120,28 @@ TEST(BitUtil, BitsetStack) {
 // test the basic assumption of word level Bitmap::Visit
 TEST(Bitmap, ShiftingWordsOptimization) {
   // single word
+  // this test assumes the little endian memory layout in *bytes
+  // Therefore, convert data layout of word to little-endian on big-endian platform
   {
     uint64_t word;
     auto bytes = reinterpret_cast<uint8_t*>(&word);
     constexpr size_t kBitWidth = sizeof(word) * 8;
 
     for (int seed = 0; seed < 64; ++seed) {
       random_bytes(sizeof(word), seed, bytes);
+      uint64_t original_word = word;
```

Comment:
Wouldn't it be simpler to write `uint64_t native_word = FromLittleEndian(word)`?

Reply:
Thank you for your great suggestion. Now, it is much simpler.